apm-sniffer/apm-sdk-plugin/hystrix-1.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/hystrix/v1/HystrixConcurrencyStrategyInterceptor.java

```diff
@@ -20,12 +20,18 @@
 package org.apache.skywalking.apm.plugin.hystrix.v1;
 
 import com.netflix.hystrix.strategy.concurrency.HystrixConcurrencyStrategy;
-import java.lang.reflect.Method;
+
 import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.EnhancedInstance;
 import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.InstanceMethodsAroundInterceptor;
 import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.MethodInterceptResult;
 
+import java.lang.reflect.Method;
+import java.util.concurrent.atomic.AtomicReference;
+
 public class HystrixConcurrencyStrategyInterceptor implements InstanceMethodsAroundInterceptor {
+
+    private final AtomicReference<SWHystrixConcurrencyStrategyWrapper> wrapper = new AtomicReference<SWHystrixConcurrencyStrategyWrapper>();
```

Comment:
This filed should store in `EnhancedInstance` object , Not in interceptor object.

Reply:
@chenpengfei Let me introduce more about the differences between attribute in here and `EnhancedInstance#dynamicField`.

The Interceptor is the singleton for each classloader, `EnhancedInstance#dynamicField` is the real attachment for the Java instance. Choose what you need.

In most case, I agree with @ascrutae , `in interceptor object` is not a good choice. :)

Reply:
Got it. I create new PR #1610 to replace this.
New PR will fix memory leak of SWExecutionHookWrapper and SWHystrixConcurrencyStrategyWrapper together.